pyfolio/timeseries.py

```diff
@@ -584,6 +586,55 @@ def rolling_multifactor_beta(returns, df_multi_factor, rolling_window=63):
     return out
 
 
+def rolling_risk_factors(returns, risk_factors=None,
+                         rolling_beta_window=63 * 2):
```

Comment:
Delaney,
The 63 * 2 number is a bit of an artifact from the original prototype
code.  It's just supposed to be reflective of a 6-month window of trading
days (e.g. 21 * 6). we should probably just swap in the value 126 which is
the approx # of trading days in a 6-month period.  I'm also totally open to
other more robust approaches.


On Fri, Aug 28, 2015 at 9:03 AM, Delaney Granizo-Mackenzie <
notifications@github.com> wrote:

> In pyfolio/timeseries.py
> <https://github.com/quantopian/pyfolio/pull/130#discussion_r38215262>:
>
> > @@ -584,6 +586,55 @@ def rolling_multifactor_beta(returns, df_multi_factor, rolling_window=63):
> >      return out
> >
> >
> > +def rolling_risk_factors(returns, risk_factors=None,
> > +                         rolling_beta_window=63 * 2):
>
> Where does 63 * 2 come from?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/quantopian/pyfolio/pull/130/files#r38215262>.
>